src/platform_impl/web/stdweb/mod.rs

```diff
@@ -41,6 +41,14 @@ pub fn window_size() -> LogicalSize {
     LogicalSize { width, height }
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio
+// TODO: Use media queries to register changes in dpi: https://jsfiddle.net/b6zcg24u/
+// TODO: Where does winit handle DPI changes? we can resize the "backbuffer" (canvas element), but isn't that usually handled by e.g. gfx?
```

Comment:
I think changing the "backbuffer" and emitting a DPI change event is sufficient here.